<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doubleAdjust</class>
 <widget class="QWidget" name="doubleAdjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btn_ok_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:6px;
color: rgb(255, 255, 255);
border:2px solid rgba(255, 255, 255,70);
background-color: rgba(255, 255, 255, 20);
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>第一点期望值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:6px;
color: rgb(255, 255, 255);
border:2px solid rgba(255, 255, 255,70);
background-color: rgba(255, 255, 255, 20);
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;</string>
   </property>
   <property name="text">
    <string>校   准</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/1.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>151</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>当前值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_current">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:6px;
color: rgb(255, 255, 255);
border:2px solid rgba(255, 255, 255,70);
background-color: rgba(255, 255, 255, 20);
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>600</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:6px;
color: rgb(255, 255, 255);
border:2px solid rgba(255, 255, 255,70);
background-color: rgba(255, 255, 255, 0);
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;</string>
   </property>
   <property name="text">
    <string>取   消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>第二点期望值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:6px;
color: rgb(255, 255, 255);
border:2px solid rgba(255, 255, 255,70);
background-color: rgba(255, 255, 255, 20);
border-top-right-radius:6px;
border-bottom-left-radius:6px;
border-bottom-right-radius:6px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>lineEdit_current</zorder>
  <zorder>btn_ok_3</zorder>
  <zorder>label_4</zorder>
  <zorder>btn_ok_2</zorder>
  <zorder>label_3</zorder>
  <zorder>btn_ok_4</zorder>
  <zorder>label_5</zorder>
  <zorder>btn_ok_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
